tests/link/jax/test_basic.py

```diff
@@ -27,9 +27,12 @@ def set_aesara_flags():
 jax = pytest.importorskip("jax")
 
 
-opts = RewriteDatabaseQuery(include=["jax"], exclude=["cxx_only", "BlasOpt"])
```

Comment:
Use separate queries for Python and JAX mode so `JAXShapeTuple` are only present in the graphs compiled by JAX.

Reply:
I added a commit to that effect.